sherpa/cpp_api/grpc/online-grpc-client-impl.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 Ximalaya Speech Team (Xiang Lyu)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GRPC_GRPC_CLIENT_H_
+#define GRPC_GRPC_CLIENT_H_
+
+#include <iostream>
+#include <memory>
+#include <string>
+#include <thread>
+
+#include "grpc/grpc.h"
+#include "grpcpp/channel.h"
+#include "grpcpp/client_context.h"
+#include "grpcpp/create_channel.h"
+
+#include "sherpa/csrc/log.h"
+#include "sherpa.grpc.pb.h"
```

Comment:
```suggestion
#include "sherpa/cpp_api/grpc/sherpa.grpc.pb.h"
```

Reply:
since "sherpa.grpc.pb.h" is generated by protoc and we change the generate path from CMAKE_CURRENT_SOURCE_DIR to CMAKE_CURRENT_BINARY_DIR,  and now it exist in build path.

Reply:
i add include_directories(${CMAKE_CURRENT_BINARY_DIR}) in grpc cmake to include sherpa.grpc.pb.h

Reply:
The code style is to use
```
#include "a/b/c.h"
```
instead of
```
#include "c.h"
```

even if `#include "c.h"` does not cause compilation errors.

It is just for following the code style.

Reply:
ok, get it

Reply:
I suggest that you use
https://github.com/k2-fsa/sherpa/blob/master/scripts/check_style_cpplint.sh
to check your code style.

```bash
cd /path/to/sherpa
./scripts/check_style_cpplint.sh 2
```

Reply:
ok ,  i will use it and pr today